HotelReservation/Api/Startup.cs

```diff
@@ -45,11 +69,18 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseRouting();
 
             app.UseAuthorization();
+            app.UseAuthentication();
```

Comment:
app.UseAuthorization();
app.UseAuthentication();
is it the right order?

Reply:
my bad, already replaced.